src/components/ModalPayment/ModalPayment.tsx

```diff
@@ -32,11 +32,14 @@ const ModalPayment = ({
   const payment: PaymentParams = {
     amount: Number(donatedAmt) * 100,
     currency: 'usd',
-    item_type: purchaseType === 'donation' ? 'donation' : 'gift_card',
+    item_type: purchaseType,
     quantity: 1,
     seller_id: sellerId,
   };
 
+  const purchaseTypePhrase =
+    purchaseType === 'donation' ? 'Donation' : 'Gift card purchase';
```

Comment:
I would make donation and gift card lower case since they aren't proper nouns

Reply:
Or if you prefer them as proper nouns, capitalize Card as well. 

So either: donation, gift card
or
Donation, Gift Card